statsmodels/genmod/tests/results/res_R_var_weight.py

```diff
@@ -0,0 +1,8615 @@
+# This file auto-generated by test_get_R_tweedie_var_weight.R
+# Using: R version 3.2.4 Revised (2016-03-16 r70336)
+import numpy as np
+
+
+class Bunch(dict):
+    def __init__(self, **kw):
+        dict.__init__(self, kw)
+        self.__dict__  = self
+
+res = dict() 
+res['params'] = np.array([0.74699331283645698, -0.031592165334821391, -0.021959412079721258,
+ ])
+res['bse'] = np.array([0.25438368514035242, 0.011629899831763216, 0.011034604276375988,
+ ])
+res['deviance'] = 45399.238887 
+res['ll'] = np.nan 
+res['resids_colnames'] = ['resid_response', 'resid_pearson', 'resid_deviance', 'resid_working'] 
+res['resids'] = np.array([-0.51916916931427282, 2.5547051833524539, 0.40292815811196381,
+ 0.27080945814608781, 3.9285319711191033, 3.9285319711191033,
```

Comment:
OK. Initially it was only 20 lines because there were no line breaks. 

I'll make a csv.